x-pack/plugins/ml/public/alerting/job_selector.tsx

```diff
@@ -91,11 +108,24 @@ export const JobSelectorControl: FC<JobSelectorControlProps> = ({
               },
             ]
           : []),
+
         {
           label: i18n.translate('xpack.ml.jobSelector.jobOptionsLabel', {
             defaultMessage: 'Jobs',
           }),
-          options: jobIdOptions.map((v) => ({ label: v })),
+          options: [
+            ...(createJobUrl
+              ? [
+                  {
+                    label: i18n.translate('xpack.ml.jobSelector.createNewLabel', {
+                      defaultMessage: '---Create new---',
```

Comment:
Nit - I'd add whitespace chars between the `Create new`  and `---` here, as is done here for example

<img width="284" alt="image" src="https://github.com/elastic/kibana/assets/7405507/3be465c6-122f-45c7-9aee-7564f134d62b">

Reply:
Updated in https://github.com/elastic/kibana/pull/174016/commits/1861ce50b6412d4cf9e95b3b0338844e1cc5e2f1